functions/package.json

```diff
@@ -750,7 +757,7 @@
         {
           "path": "^/usermgmt/api/me/is_token_valid_internal",
```

Comment:
`is_token_valid_internal ` was previously used in perseus and not used anymore. changed it to researcher scope and used for ws validation